databricks/koalas/tests/test_series_plot.py

```diff
@@ -265,11 +265,30 @@ def test_box_summary(self):
         self.assert_eq(expected_whiskers[1], whiskers[1])
         self.assert_eq(expected_fliers, fliers)
 
-    def test_missing(self):
-        ks = self.kdf1['a']
-
-        unsupported_functions = ['kde']
-        for name in unsupported_functions:
-            with self.assertRaisesRegex(PandasNotImplementedError,
-                                        "method.*Series.*{}.*not implemented".format(name)):
-                getattr(ks.plot, name)()
+    def test_kde_plot(self):
+        pdf = self.pdf1
+        kdf = self.kdf1
+
+        kax = pdf['a'].plot("kde", colormap='Paired')
```

Comment:
btw, do you mind swapping the `ax` name here... since this seems like `pax` and below should be `kax`

Reply:
@charlesdong1991 Oops you're right. I made a mistake. Thanks!